run.py

```diff
@@ -13,29 +14,16 @@
 the given module or script.
 """
 
-import os
-import sys
 import imp
```

Comment:
There are also some other things pylint complains about, so you may want to fix some of this in one go.